files/default/sql/ddl/updates/1.4.0.sql

```diff
@@ -1,42 +1,36 @@
-ALTER TABLE `hopsworks`.`python_dep` ADD COLUMN `base_env` VARCHAR(45) COLLATE latin1_general_cs;
-
-TRUNCATE TABLE `hopsworks`.`conda_commands`;
--- drop foreign key to project it is not always pointing to a project now.
-SET @fk_name = (SELECT CONSTRAINT_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE TABLE_SCHEMA = "hopsworks" AND TABLE_NAME = "conda_commands" AND REFERENCED_TABLE_NAME="project");
-SET @s := concat('ALTER TABLE hopsworks.conda_commands DROP FOREIGN KEY `', @fk_name, '`');
-PREPARE stmt1 FROM @s;
-EXECUTE stmt1;
-DEALLOCATE PREPARE stmt1;
-
 SET @fk_name = (SELECT CONSTRAINT_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE TABLE_SCHEMA = "hopsworks" AND TABLE_NAME = "conda_commands" AND REFERENCED_TABLE_NAME="hosts");
 SET @s := concat('ALTER TABLE hopsworks.conda_commands DROP FOREIGN KEY `', @fk_name, '`');
 PREPARE stmt1 FROM @s;
 EXECUTE stmt1;
 DEALLOCATE PREPARE stmt1;
 
 ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `host_id`, DROP INDEX `host_id` ;
-ALTER TABLE `hopsworks`.`conda_commands` CHANGE `proj` `docker_image` varchar(255) COLLATE latin1_general_cs NOT NULL;
+ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `proj`;
+ALTER TABLE `hopsworks`.`conda_commands` CHANGE `environment_yml` `environment_yml` VARCHAR(1000) COLLATE latin1_general_cs DEFAULT NULL;
```

Comment:
Isn't it `1000` too low for a whole yaml file?

Reply:
Please don't shoot the messenger. I just refactored what was there. 

@tkakantousis, @robzor92 : i just exported the yml file for the base environment and it's 5260 chars long.

Reply:
we don't the actual yml, we store the path to the yaml in hdfs

Reply:
https://github.com/logicalclocks/hopsworks/blob/2ddd8bfc8449698a87ef46dda56eb4c8bb6ca2a4/hopsworks-common/src/main/java/io/hops/hopsworks/common/python/environment/EnvironmentController.java#L233